uikit/common/sdk/src/main/java/com/sygic/ui/common/sdk/mapobject/OnClickMapMarker.kt

```diff
@@ -0,0 +1,7 @@
+package com.sygic.ui.common.sdk.mapobject
+
+import androidx.annotation.RestrictTo
+import com.sygic.sdk.map.`object`.ViewObject
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+class OnClickMapMarker(viewObject: ViewObject) : MapMarker(viewObject)
```

Comment:
aku ulohu ma mat ten `OnClickMapMarker` oproti `MapMarker`?

Reply:
je restricted na použití v library a potom ho dokáži jednoduše odlišit v tom manageru bez toho, aniž bych musel mýt uložený nějaký vnitřní typ ;)